### ISSUE TITLE
generate_template.py doesn't create iso file

### ISSUE THREAD

**Issue by dgranville** [elik1001/coreos-kubernetes-generator#2]
generate_template.py claims to create a file called <hostname>_template.iso, but no such file exists in the configs folder.  Below is the actual output from the command.

```
Ignition Template created successfully.
-----------------------------------------------------------
  SSH private key is             : keys/id_rsa
  SSH public key is              : keys/id_rsa.pub
  SSL certificates are in        : ./ssl/
  Build Template file is         : configs/resource_template.yaml
  Ignition Template file is      : configs/resource_template.ign
  Ignition Template ISO file is  : configs/resource_template.iso
```
and this is everything in that directory:
```
# ls -a configs
.  ..  resource_template.ign  resource_template.yaml
```

**Comment by elik1001** [comment]
I am assuming you are missing the mkisofs binary.

On redhat try to run.
yum install genisoimage
On Ubuntu try.
apt-get install mkisofs

Thanks for the feedback, rely appreciated, I will update the script to
verify / install if required.  However, please note that the script still
has a number of issues especially with worker nodes, i am about to push out
an update hopefully somewhere tomorrow.

The update will also change the ip address ranges, as the current setup has
a number of issues.

Again, thanks for your input
Eli

On Dec 14, 2017 6:34 PM, "Daniel Granville" <notifications@github.com>
wrote:

generate_template.py claims to create a file called _template.iso, but no
such file exists in the configs folder. Below is the actual output from the
command.

Ignition Template created successfully.
-----------------------------------------------------------
  SSH private key is             : keys/id_rsa
  SSH public key is              : keys/id_rsa.pub
  SSL certificates are in        : ./ssl/
  Build Template file is         : configs/resource_template.yaml
  Ignition Template file is      : configs/resource_template.ign
  Ignition Template ISO file is  : configs/resource_template.iso

and this is everything in that directory:

# ls -a configs
.  ..  resource_template.ign  resource_template.yaml

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/elik1001/coreos-kubernetes-generator/issues/2>, or mute
the thread

**Comment by elik1001** [comment]
Fixed in latest build  Version 0.2, now committed to master.
